sr_moveit_hand_config/config/kinematics_template.yaml

```diff
@@ -1,3 +1,10 @@
+handpalm:
+  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
```

Comment:
is this meant to be kdl_kinematics_plugin/KDLKinematicsPlugin? or the same as the others (hand_kinematics/HandKinematicsPlugin)?